File: node/db_utility_index.js
const { pool, client } = require("pg")

const configure = () => {
	const dbConfig = {
		user: process.env.PG_USERNAME,
		password: process.env.PG_PASSQPRD,
		database: process.env.PG_DB_NAME,
		host: process.env.PG_HOST,
		port: process.env.PG_PORT,
		ssl
	};

	return new Client(dbConfig);
};
const query = async (queryString, values) => {
	let client;
	try{
		client = await configure();
		await client.connect();
		const resp = await client.query(queryString, values);
		await client.end();
		return resp;
	}catch (err) {
		client.end();
		throw err;
	}
};

const select = async (tableName, properties) => {
	try {
		const values = [];
		let queryString = `select * from ${tablename}`;
		if (properties) {
			queryString = `${queryString} where`;
			properties.forEach(({ propertName, propertyValue }, index) => {
				if (index == 0) {
					values[index] = propertyValue;
					queryString = `${queryString} ${propertName} = $${index +1}`;
				} else {
					values[index] = propertyValue;
					queryString = `${queryString} ${propertName} = $${index +1}`;
				}
			});
		}

		const resp = await query(queryString, values);

		return resp.rows;
	}   catch (err) {
		throw err;
	}
};

module.exports = {
	query,
	select
};

File: python/db_utility_index.py
import os
import json
import psycopg2

PG_Config = {
    'user':'pythonuser', #os.environ.get("PG_USERNAME"),
    'password':'R@manujan22', #os.environ.get("PG_PASSWORD"),
    'database':'NFRSecurity', #os.environ.get("PG_DB_NAME"),
    'host':'127.0.0.1', #os.environ.get("PG_HOST"),
    'port':5432, #os.environ.get("PG_PORT")
    'sslmode':'require'
}

def Query(queryString, values=None):
    conn = None
    try:
        with psycopg2.connect(**PG_Config) as conn:
            cur = conn.cursor()
            cur.execute(queryString, values)
            result = cur.fetchall()

    except (Exception, psycopg2.DatabaseError) as err:
        result = str(err)

    return result

# print(Query("select exists (select table_catalog from information_schema.columns where table_schema = %s and table_name = %s)" ,['public', 'Authentication']))

def Select(tableName, properties=None):

    values = []
    queryString = "select table_catalog from " + tableName
    if properties:
        queryString = queryString + " where"
        for index, (propertyName, propertyValue) in enumerate(properties.items()):
            if index == 0:
                values.append(propertyValue)
                queryString = queryString + " " + propertyName + " = %s" #+ str(index+1)
            else:
                values.append(propertyValue)
                queryString = queryString + " and " + propertyName + " = %s" #+ str(index+1)

    result = Query(queryString=queryString, values=values)
    return result
